### DISCUSSION TITLE
How to fix VipsForeignSave: "test.webp" is not a known file format

### DISCUSSION THREAD

**Discussion by pollinoco** [libvips/libvips#3500] [Q&A]
App Error:
VipsForeignSave: "/tmp/image_processing20230519-424119-c3o0pc.webp"is not a known file format.....

Local is fine!
But  in my Ubuntu server try:

vips copy image.jpg test.webp
           
          VipsForeignSave: "test.webp" is not a known file format

vips --vips-config
          
          ......
          HEIC/AVIF load/save with libheif: no (dynamic module: no)
          **WebP load/save with libwebp: no**
          PDF load with PDFium:  no
          PDF load with poppler-glib: no (dynamic module: no)
          ......

vips -l | grep _target

          VipsForeignSaveCsvTarget (csvsave_target), save image to csv (.csv), priority=0, mono
          VipsForeignSaveMatrixTarget (matrixsave_target), save image to matrix (.mat), priority=0, mono
          VipsForeignSaveVipsTarget (vipssave_target), save image to target in vips format (.v, .vips), priority=0, any
          VipsForeignSavePpmTarget (ppmsave_target), save to ppm (.ppm), priority=0, rgb
          VipsForeignSaveRadTarget (radsave_target), save image to Radiance target (.hdr), priority=0, rgb
          VipsForeignSavePngTarget (pngsave_target), save image to target as PNG (.png), priority=0, rgba
          VipsForeignSaveJpegTarget (jpegsave_target), save image to jpeg target (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveTiffTarget (tiffsave_target), save image to tiff target (.tif, .tiff), priority=0, any

Thank you for your help!

**Comment by jcupitt** [comment]
Hi @pollinoco,

Your server's libvips has been built without WEBP support. You'll need to dig into where that binary came from, and what version it is.
